packages/cbioportal-clinical-timeline/src/TimelineTracks.tsx

```diff
@@ -9,6 +9,7 @@ import { flattenTracks, sortNestedTracks } from './lib/helpers';
 import CustomTrack, { CustomTrackSpecification } from './CustomTrack';
 import { TICK_AXIS_HEIGHT } from './TickAxis';
 import { useObserver } from 'mobx-react-lite';
+import { ToolTipModel } from '../../../src/pages/studyView/charts/barChart/BarChartToolTip';
```

Comment:
i'm pretty sure this is the problem.  i have a branch building to test.   1) this import is not used. 2) packages shouldn't import from main project